docs/community.md

```diff
@@ -64,103 +27,43 @@ Anyone interested in spatial data infrastructures, libraries, GIS, maps, data cu
 
-## Community Roles
-
-### Administrative Roles
-
-**Meeting Facilitators**: Becky Seifried (July 2025–July 2026)
-
-* Manage Google calendar
-* Set up and host monthly meetings 
-* Prepare agendas
-
-**Platform Management Team**: ==2 vacancies==
-
-* Manage GitHub users and roles
-* Manage Google group permissions, roles, etc.
-* Manage Google Drive members
-* Maintain Slack 
-
-### Documentation Roles
-
-**Public Content Manager**: ==1 vacancy==
+ - ### Help Triage Issues
+  
+    Join our Issue Triage Group, where developers meet quarterly to review and prioritize GitHub issues. [Reach out on Slack to join.](https://join.slack.com/t/geo4lib/shared_invite/zt-3baz35i9x-gLVsSGz8MCDlkGePjk7Ejw)
 
-* Keep the public website(s) up to date
-* Write periodic blog posts, including sprint wrap-up posts
+    [:material-github: Review our ongoing project board](https://github.com/orgs/geoblacklight/projects/2) 
 
-**Technical Writing Lead**: ==1 vacancy==
+</div>
 
-* Keep track of what code, metadata, and tutorials need updating
-* Manage GitHub issues related to documentation
-* Coordinate volunteers to update documentation as necessary
 
-### Technical Roles
+## GeoBlacklight Project Maintainers
 
-**Designated Code Mergers**: [see GitHub Teams](https://github.com/orgs/geoblacklight/teams)
+**Designated Code Mergers**: Nick Budak + [see GitHub Teams](https://github.com/orgs/geoblacklight/teams)
 
-* Members of the GeoBlacklight Developers team in GitHub
-* Review pull requests on behalf of the team, or assign individual code reviewers based on expertise  
+* Review pull requests
+* Assign individual code reviewers based on expertise  
 
-**Release Coordinators**: Eric Larson + ==1 vacancy==
+**Release Coordinators**: Eric Larson
 
 * Cut tags and releases in the GitHub repository, and write release notes
 * Publish the GeoBlacklight [Ruby gem](https://rubygems.org/gems/geoblacklight)
 * Publish Javascript to [npm](https://www.npmjs.com/package/@geoblacklight/frontend) 
 
-## Workgroups and Interest Groups
-
-<div class="grid cards" markdown>
-
-- ### Issue Triage Workgroup
-
-	Meets quarterly to review and prioritize GitHub issues related to code.
-	
-	* Eliot Jordan
-	* Eric Larson
-	* Karen Majewicz
-	* Keith Jenkins
-	* Maura Carbone
-
-- ### Governance Workgroup
-
-	Improves our communication around change management, connected projects, and community roles.
-	
-	* Karen Majewicz
-	* Becky Seifried
-	* Eliot Jordan
-
-- ### Georeferencing Interest Group
-
-	Discusses georeferencing projects and strategies to enhance GeoBlacklight’s georeferenced map capabilities.
-	
-	* Adam Cox
-	* Alex Marden
-	* Eliot Jordan
-	* Kim Leaman
-	* Marc McGee
-	* Michael Shensky
-	* Stephen Appel
-
-
-</div> 
+**Technical Writing Lead**: ==1 vacancy==
 
+* Keep track of what code, metadata, and tutorials
+* Manage GitHub issues related to documentation
+* Coordinate volunteers to update documentation as needed
 
 
```

Comment:
Do we need a dedicated GeoBlacklight "Platform Manager" to manage the GBL GitHub users and roles? Or would this duty be part of the Geo4Lib platform management team?